django_mongodb/client.py

```diff
@@ -1,14 +1,51 @@
 import signal
+import urllib
 
 from django.db.backends.base.client import BaseDatabaseClient
 
 
 class DatabaseClient(BaseDatabaseClient):
-    executable_name = "mongo"
+    executable_name = "mongosh"
+    help_options = ("--help", "-h")
 
     @classmethod
     def settings_to_cmd_args_env(cls, settings_dict, parameters):
-        raise NotImplementedError
+        options = settings_dict.get("OPTIONS", {})
+        args = [cls.executable_name]
+
+        host = settings_dict["HOST"]
+        port = settings_dict["PORT"]
+        dbname = settings_dict["NAME"] or "test"
+        user = settings_dict["USER"]
+        passwd = settings_dict["PASSWORD"]
+        auth_database = options.get("authSource")
+        auth_mechanism = options.get("authMechanism")
+        retry_writes = options.get("retryWrites")
+
+        if host:
+            args += ["--host", host]
+        if port:
+            args += ["--port", port]
+        if user:
+            args += ["--username", user]
+        if passwd:
+            passwd = urllib.parse.quote_plus(passwd)
```

Comment:
Did you read a recommendation to use quote_plus() somewhere? I can understand spaces in the password might be a problem on the command line, but it seems questionable that percent escaping special characters is the solution.